### PR TITLE
Nithin

### PR DESCRIPTION
**Pull request by nithindeva**
iam merging files in nithin branch to dummy.please review it.

### CODE REVIEW

File: project_devops/gitbasics.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+git commit -a -m
+git add -A
+git remote add origin
+git checkout
+git branch
+git merge
+git pull origin master
+git pull origin branchname
+git log
+git --help
```
Comment (david1412) [inline project_devops/gitbasics.txt]: Could you please add the new commands what you learnt yesterday.
Commit and create PR again.

**Review by david1412 — COMMENTED** [review]
Please do this task with Prudhvi, this is a team activity. I have left one comment.

File: project_devops/gitbasics.txt (hunk 1 above)
Comment (nithindeva) [reply project_devops/gitbasics.txt]: okay

Comment (david1412) [reply project_devops/gitbasics.txt]: Still, the explanation is not clear to me. See dummy where Phani and Pradeep worked on this task.

**Review by david1412 — COMMENTED** [review]
See the comments.
+ updated the GitHub and Linux file with an explanation. what is the usage of the command?
+ Scrum file is missing. Please add this file with some explanation.

File: temp/gitbasics.txt [hunk 2]
```diff
@@ -0,0 +1,29 @@
+1.git config --global user.name "username"
+2.git config --global user.email "email"
+3.git init
```
Comment (david1412) [inline temp/gitbasics.txt:8]: Still, the task is not complete.
Example: git init > It creates a .git file, It is used in backend to switch the branch, store the changes in local and remote, etc.

File: temp/gitbasics.txt [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+4.git clone
+5.git status
+6.git add
+7.git push -u origin master   or git push origin branchname
+8.git pull
```
Comment (david1412) [inline temp/gitbasics.txt]: Example: git pull > Pull the current changes in the origin/branch. This command will update the current commits.

Usage is missing, please update with an explanation.

**Review by david1412 — COMMENTED** [review]
Please update the commands with an explanation.

File: temp/gitbasics.txt (hunk 3 above)
Comment (nithindeva) [reply temp/gitbasics.txt]: OKAY I GOT IT.

**Review by david1412 — APPROVED** [review]
Perfect! nice work

**Pull request merged**